src/pages/K8s.vue

```diff
@@ -201,6 +201,14 @@ export default {
       console.log('settings have been updated');
       this.$data.settings = settings;
     });
+    (async() => {
```

Comment:
I think we should minimize our uses of `ipcRenderer.invoke` in the UI, and instead break it into a `send` and an `on`, like the use of `images-namespaces` and `images-namespaces-read` at https://github.com/rancher-sandbox/rancher-desktop/blob/f3d631a161ac684983b81329781ce5c44ddc9acf/src/pages/Images.vue#L95  (but please wait for Mark's opinion).